### DISCUSSION TITLE
Xcode 14.3 Scope Cannot infer key path type from context

### DISCUSSION THREAD

**Discussion by mkelowna** [pointfreeco/swift-composable-architecture#2016] [Q&A]
I have just updated to Xcode 14.3 and am seeing breaking changes to this code peppered throughout my app:

    Scope(state: \.dragnDrop,
              action: /Action.dragnDrop) {
            DragnDrop()
        }

I am getting the following errors:

1. Cannot infer key path type from context; consider explicitly specifying a root type, and
2. Generic parameter 'Child' could not be inferred

Any help would be appreciated.

**Comment by tgrapperon** [comment]
Hey @mkelowna! Sorry for not responding sooner. There were a lot of changes with the way result builder are working in Swift 5.8, and the library had to be updated. Please make sure that you're running at least TCA v0.50.0 (ideally v0.52.0).
